spec/controllers/brapi/v1/studies_controller_spec.rb

```diff
@@ -0,0 +1,167 @@
+require 'rails_helper'
+require 'pp'
+
+RSpec.describe Brapi::V1::StudiesController do
+  
+  context 'BrAPI without requiring authentication' do
+    let(:u1) { create :user }
+    before { sign_in u1 }
+    
+    describe '#search' do
```

Comment:
These controller specs are now redundant and should be removed. However, some of the specs for studies API fail, please check what's wrong. Other than that, I think you can merge.